README.md

```diff
@@ -75,6 +75,9 @@ eight_points_guzzle:
     # (de)activate logging/profiler; default: %kernel.debug%
     logging: true
 
+    # configure when a response is cosidered to be slow (in ms); default 500
```

Comment:
I'm not sure about 500 as default value. Something between 50 and 500 is really good result. Do we want to announce users by default when something is not so good, or let's inform when request is really bad?

@florianpreusner what do you think?

Reply:
yeah, hard to define cause it really depends on the system. sometimes even 2 seconds are totally fine when retrieving a lot of data (from a ERP system for example). Maybe it should be 0 (deactivated) by default?

Reply:
0 for deactivated sounds like a good default to me.